Remotion/Web/Development.WebTesting/Schemas/WebTestingConfiguration.xsd

```diff
@@ -27,6 +27,7 @@
       <xs:attribute name="webApplicationRoot" type="c:RequiredStringType" use="required" />
       <xs:attribute name="screenshotDirectory" type="c:RequiredStringType" />
       <xs:attribute name="logsDirectory" type="c:RequiredStringType" default="." />
+      <xs:attribute name="chromeDisableSecurityWarningsBehavior" type="c:ChromeDisableSecurityWarningsBehavior" default="Fail" />

```

Comment:
should the default be fail or ignore? with ignore as the default, we wouldn't change the existing behavior, correct? The only downside is that a user would need to check the documentation. On the other hand, users that don't use screenshots beyond the "error screenshot" wouldn't be affected from using the existing behavior. To me, this means we should use the non-breaking-change value as default.